decidim-core/app/helpers/decidim/scopes_helper.rb

```diff
@@ -62,10 +62,18 @@ def scopes_picker_tag(name, value, options = {})
     # Renders a scopes picker field in a filter form.
     # form - FilterFormBuilder object
     # name - attribute name
+    # checkboxes_on_top - Show picker values on top (default) or below the picker prompt
```

Comment:
This modifies all other scope pickers, right? E.g. in meetings, budgets, proposals... Did you check everything looks good in those places too?

Reply:
Thanks, I missed one helper that it's not being used, but if were it would have been bad.

`Decidim::DecidimFormHelper::scopes_picker_field_tag` "picker field constructor" doesn't receive the `checkboxes_on_top` option from the helper that calls it `Decidim::ScopesHelper::scopes_picker_tag`, instead it has it hardcoded to true (used nowhere).

`Decidim::FormBuilder.scopes_picker` is the "picker field constructor" used everywhere, it receives `checkboxes_on_top` options from the helpers that call it, which have it true by default:
- `Decidim::ScopesHelper.scopes_picker_filter` (calculates the options automatically)
- `Decidim::ScopesHelper.scopes_picker_field` (calculates the options automatically)
- `Decidim::FilterFormBuilder.scopes_picker` (used nowhere; needs to receive all options)

Reply:
Wait, @mrcasals, let me review the scopes helpers one more time...